scripts/ray/dev_tpu.py

```diff
@@ -64,9 +64,9 @@
 
 # N.B. We have to import from "src" as we are using the `ray.remote` directly from our `uv run`
 # script instead of launching a driver job. This confuses cloudpickle for some reason.
-from src.marin.cluster import ray as ray_utils
```

Comment:
Ack, this one is on purpose due to a weirdness in Ray -- see the comment above. The import path has to match the file path for some reason. (If there's a fix for cloudpickle happy to use that instead.)

Reply:
oh crap sorry